store/migration/dev/LATEST.sql

```diff
@@ -170,7 +170,8 @@ CREATE TABLE project (
     db_name_template TEXT NOT NULL,
     role_provider TEXT NOT NULL CHECK (role_provider IN ('BYTEBASE', 'GITLAB_SELF_HOST', 'GITHUB_COM')) DEFAULT 'BYTEBASE',
     schema_version_type TEXT NOT NULL CHECK (schema_version_type IN ('TIMESTAMP', 'SEMANTIC')) DEFAULT 'TIMESTAMP',
-    schema_change_type TEXT NOT NULL CHECK (schema_change_type IN ('DDL', 'SDL')) DEFAULT 'DDL'
+    schema_change_type TEXT NOT NULL CHECK (schema_change_type IN ('DDL', 'SDL')) DEFAULT 'DDL',
+    lgtm_check JSONB NOT NULL DEFAULT '{}' 
```

Comment:
If for extensibility, what if we create a general payload field?

Reply:
A general payload field might be way too generic IMO.
If we put all project settings into a single object,
```javascript
payload = {	
	settingA: {
		value: 1,
    },
	settingB: {...many fields},
	settingC: {...many fields},
}
```

To upsert might be something like
```javascript
payloadPatch = {	
	settingA: {
		value: 2,
    },
	settingB: {...many fields},
	settingC: {...many fields},
}
```

It may be too troublesome to modify a field in a javascript object in codes.